pset/updater_test.go

```diff
@@ -0,0 +1,71 @@
+package pset
+
+import (
+	"encoding/hex"
+	"github.com/stretchr/testify/assert"
+	"github.com/vulpemventures/go-elements/transaction"
+	"testing"
+)
+
+func TestUpdaterAddInput(t *testing.T) {
+	inputs := make([]*transaction.TxInput, 0)
+	output := make([]*transaction.TxOutput, 0)
+	pset, err := New(inputs, output, 2, 0)
```

Comment:
please, call this `p`, since pset is the name of the package.

Reply:
Done